configs/10.0/packages/liburcu/sources

```diff
@@ -21,7 +21,7 @@
 
 ATSRC_PACKAGE_NAME="Userspace RCU"
 ATSRC_PACKAGE_VER=0.9.4
```

Comment:
Update to 0.9.5.  Source: http://git.lttng.org/?p=userspace-rcu.git;a=commit;h=5f6503c78ee929cd77d6e51987b5826de5a5fe99